openedx/core/djangoapps/auth_exchange/tests/test_forms.py

```diff
@@ -38,7 +38,6 @@ def _assert_error(self, data, expected_error, expected_error_description):
             form.errors,
             {"error": expected_error, "error_description": expected_error_description}
         )
-        self.assertNotIn("partial_pipeline", self.request.session)
```

Comment:
`partial_pipeline` is no longer available in session. we only have `partial_pipeline_token` which already present even if error occurs